lib/core/src/Cardano/Wallet/Primitive/Types/TokenMap.hs

```diff
@@ -213,6 +215,8 @@ instance PartialOrd TokenMap where
         (getAssets m1 `Set.union` getAssets m2)
 
 instance NFData TokenMap
+instance Hashable TokenMap where
+    hashWithSalt = hashUsing (Map.toList . unTokenMap)
```

Comment:
Can't `GeneralizedNewtypeDeriving` or `DerivingVia` handle that just fine :thinking: ?

Reply:
It didn't seem to work - I think perhaps Map didn't have a Hashable instance.